client/src/components/Ticketing/ticketingmanager/Season/components/SeasonInfo.tsx

```diff
@@ -118,6 +96,8 @@ const SeasonInfo = (
         success: true,
         handleClose: () => setShowPopUp(false),
         handleProceed: () => setShowPopUp(false),
+        showSecondary: false,
+        showClose: false,
```

Comment:
Do you think you could have this change for this modal?
<img width="837" alt="image" src="https://github.com/WonderTix/WonderTix/assets/92124260/f32b76f6-b36e-4653-b23a-7cf0346cb8b1">